### ISSUE TITLE
CURIE with invalid scheme rather than prefixed URL

### ISSUE THREAD

**Issue by VladimirAlexiev** [OP-TED/model2owl#139]
ePO_restrictions.ttl has 66 occurrences of `<skos:Concept>`, eg 
```ttl
epo:appliesRemedyType         rdfs:range  <skos:Concept> .
```

Please note that `<skos:Concept>` is a CURIE using the undefined/invalid scheme `skos:`,
rather than the prefixed URL `skos:Concept`.

This is a serious mistake. Luckily it's the only one of this kind, as can be checked with
```
grep -P '<(?!http|/?u>|/?b>)' *.ttl | less
```

**Comment by andreea-pasare** [comment]
After a quick look, I noticed that [ePO_shacl_shapes.ttl](https://github.com/OP-TED/ePO/blob/master/implementation/ePO/shacl_shapes/ePO_shacl_shapes.ttl) has a similar issue, but with the usage of `<skos:inScheme>`.

Note that this happens only in the ttl output, while the rdf output looks ok. 
Recommendation to check the rdf2ttl transformation function.

**Comment by Dragos0000** [comment]
This was fixed in the new release 2.0.0.
```
            owl:unionOf ( :GreenProcurement :InnovativeProcurement :SocialProcurement ) ] ;
    rdfs:range skos:Concept .
```
